syntax = "proto3";

package broker_registration;

// Define the service for broker registration
service BrokerRegistrationService {
  // Register a broker with the server
  rpc RegisterBroker (BrokerRegistrationRequest) returns (BrokerRegistrationResponse) {}
}

// Request message for broker registration
message BrokerRegistrationRequest {
  string broker_id = 1;      // Unique ID for this broker
  string broker_name = 2;    // Display name for this broker
}

// Response message for broker registration
message BrokerRegistrationResponse {
  bool success = 1;
  string message = 2;
} 